rust/send_http_post_rust/.gitignore

```diff
@@ -0,0 +1,24 @@
+# Various IDEs and Editors
```

Comment:
Same question about `rust/send_http_post/` vs `rust/send_http_post_rust/`

Reply:
Good call. They were removed on local branch, but changes were not pushed up. Pushed up now.